package.json

```diff
@@ -82,86 +82,86 @@
+    "@babel/eslint-parser": "7.21.8",
+    "@babel/eslint-plugin": "7.19.1",
     "@babel/plugin-proposal-class-properties": "7.18.6",
-    "@babel/plugin-proposal-decorators": "7.18.6",
+    "@babel/plugin-proposal-decorators": "7.21.0",
     "@babel/plugin-proposal-do-expressions": "7.18.6",
-    "@babel/plugin-proposal-export-default-from": "7.18.6",
-    "@babel/plugin-proposal-export-namespace-from": "7.18.6",
-    "@babel/plugin-proposal-function-bind": "7.18.6",
+    "@babel/plugin-proposal-export-default-from": "7.18.10",
+    "@babel/plugin-proposal-export-namespace-from": "7.18.9",
+    "@babel/plugin-proposal-function-bind": "7.18.9",
     "@babel/plugin-proposal-function-sent": "7.18.6",
     "@babel/plugin-proposal-json-strings": "7.18.6",
-    "@babel/plugin-proposal-logical-assignment-operators": "7.18.6",
+    "@babel/plugin-proposal-logical-assignment-operators": "7.20.7",
     "@babel/plugin-proposal-nullish-coalescing-operator": "7.18.6",
     "@babel/plugin-proposal-numeric-separator": "7.18.6",
-    "@babel/plugin-proposal-optional-chaining": "7.18.6",
-    "@babel/plugin-proposal-pipeline-operator": "7.18.6",
+    "@babel/plugin-proposal-optional-chaining": "7.21.0",
+    "@babel/plugin-proposal-pipeline-operator": "7.18.9",
     "@babel/plugin-proposal-throw-expressions": "7.18.6",
     "@babel/plugin-syntax-dynamic-import": "7.8.3",
     "@babel/plugin-syntax-import-meta": "7.10.4",
-    "@babel/plugin-transform-classes": "7.18.6",
-    "@babel/plugin-transform-react-constant-elements": "7.18.6",
-    "@babel/plugin-transform-react-inline-elements": "7.18.6",
+    "@babel/plugin-transform-classes": "7.21.0",
+    "@babel/plugin-transform-react-constant-elements": "7.21.3",
+    "@babel/plugin-transform-react-inline-elements": "7.21.0",
     "@babel/polyfill": "7.12.1",
-    "@babel/preset-env": "7.18.6",
+    "@babel/preset-env": "7.21.5",
     "@babel/preset-react": "7.18.6",
-    "@babel/register": "7.18.6",
-    "@babel/runtime-corejs2": "7.18.6",
+    "@babel/register": "7.21.0",
+    "@babel/runtime-corejs2": "7.21.5",
     "@jest-runner/electron": "3.0.1",
     "@tidepool/direct-io": "3.0.2",
-    "aws-sdk": "2.1169.0",
+    "aws-sdk": "2.1374.0",
     "babel-core": "7.0.0-bridge.0",
     "babel-jest": "26.6.3",
     "babel-loader": "8.2.5",
     "babel-plugin-add-module-exports": "1.0.4",
     "babel-plugin-dev-expression": "0.2.2",
-    "babel-plugin-module-resolver": "4.1.0",
+    "babel-plugin-module-resolver": "5.0.0",
     "babel-plugin-rewire": "1.2.0",
-    "babel-plugin-transform-define": "2.0.1",
+    "babel-plugin-transform-define": "2.1.0",
     "babel-plugin-transform-react-remove-prop-types": "0.4.24",
-    "chai": "4.3.6",
-    "concurrently": "7.2.2",
+    "chai": "4.3.7",
+    "concurrently": "8.0.1",
     "cross-env": "7.0.3",
     "css-loader": "5.2.7",
     "difflet": "1.0.1",
-    "drivelist": "11.0.0",
-    "electron": "21.3.1",
+    "drivelist": "11.1.0",
+    "electron": "24.2.0",
```

Comment:
Does this upgrade have implications for QA load on this ticket? Seems moderately out of scope for a metrics update.

Reply:
See this comment: https://tidepool.atlassian.net/browse/UPLOAD-785?focusedCommentId=46040